src/TinyHelpers/Extensions/DateTimeExtensions.cs

```diff
@@ -39,4 +39,43 @@ public static TimeOnly ToTimeOnly(this DateTime dateTime)
     public static TimeOnly ToTimeOnly(this TimeSpan timeSpan)
         => TimeOnly.FromTimeSpan(timeSpan);
 #endif
+
+    public static string ToFriendlyDateTimeString(this DateTime Date)
+    {
+        return FriendlyDate(Date) + " @ " + Date.ToString("t").ToLower();
+    }
+
+    public static string ToFriendlyShortDateString(this DateTime Date)
+    {
+        return $"{Date.ToString("MMM dd")}, {Date.Year}";
+    }
+
+    public static string ToFriendlyDateString(this DateTime Date)
+    {
+        return FriendlyDate(Date);
+    }
+
+    static string FriendlyDate(DateTime date)
```

Comment:
Same as above. Moreover, the strings _Today_ and _Yesterday_ are provided in English only, so they can't be used in a multi-language environment. If I need to provide a friendly date, I prefer using a library like Humanizer, that fully supports localization: https://github.com/Humanizr/Humanizer#humanize-datetime.

Reply:
I added Humanizer but now the library has a dependency. Is it ok?